Sources/Gravatar/Network/Services/AvatarService.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import UIKit
+
+/// AvatarService helps with uploading and downloading avatars.
```

Comment:
```suggestion
/// A service to perform uploading and downloading of avatars.
///
/// An avatar is a profile image of a Gravatar user. See [the avatar docs](https://support.gravatar.com/profiles/avatars/) for more info.
```
To be more similar to the descriptions of `ProfileService` and `ImageDownloadService`.